Marlin/src/lcd/language/language_en.h

```diff
@@ -577,7 +586,8 @@ namespace Language_en {
   LSTR MSG_CASE_LIGHT_BRIGHTNESS          = _UxGT("Light Brightness");
   LSTR MSG_KILL_EXPECTED_PRINTER          = _UxGT("INCORRECT PRINTER");
 
-  #if LCD_WIDTH >= 20
+  #if (LCD_WIDTH >= 20 || HAS_DWIN_E3V2_BASIC)
```

Comment:
Is `LCD_WIDTH` actually under 20 –or not defined– in the case of `DWIN_E3V2_BASIC`? Maybe we can fix that.